config/routes.rb

```diff
@@ -55,6 +55,9 @@
       post '/login/two_factor/authenticator' => 'two_factor_authentication/totp_verification#create'
       get '/login/two_factor/personal_key' => 'two_factor_authentication/personal_key_verification#show'
       post '/login/two_factor/personal_key' => 'two_factor_authentication/personal_key_verification#create'
+      get '/login/two_factor/reset_device' => 'two_factor_authentication/reset_device#show'
+      post '/login/two_factor/reset_device' => 'two_factor_authentication/reset_device#create'
+      get '/login/two_factor/reset_device_cancel' => 'two_factor_authentication/cancel_reset_device#cancel'
```

Comment:
For the route /login/two_factor/reset_device_cancel I'd like to leave this as a GET request since it's a link in an email which can be a plain text email.